var/spack/repos/builtin/packages/tix/package.py

```diff
@@ -22,25 +22,25 @@ class Tix(AutotoolsPackage):
     extends('tcl', type=('build', 'link', 'run'))
     depends_on('tk', type=('build', 'link', 'run'))
 
-    patch('https://raw.githubusercontent.com/macports/macports-ports/master/x11/tix/files/panic.patch',
+    patch('https://raw.githubusercontent.com/macports/macports-ports/v2.7.0-archive/x11/tix/files/panic.patch',
```

Comment:
We should never pull patches from master, as the patch file may be changed or removed in newer versions. In fact, many of these patches are no longer available on master.